src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -61,6 +64,32 @@ export default defineComponent({
         items: {
             type: Array as PropType<DlLabelPickerItem[]>,
             default: () => [] as PropType<DlLabelPickerItem[]>
+        },
+        /**
+         * Props for the empty state component
+         */
+        emptyStateProps: {
+            type: Object as PropType<DlEmptyStateProps>,
+            default: () =>
+                ({
+                    title: '',
+                    subtitle: 'No data to show yet.',
+                    icon: 'icon-dl-dataset-filled'
```

Comment:
@yogesh0325 i think since its a label picker we dont need to show icon on empty state
was this by design ?

maybe change subtitle to `No labels to show yet.`